docs/articles/faq/README.md

```diff
@@ -259,4 +262,27 @@ and [t.resizeWindow](../documentation/test-api/actions/resize-window.md#setting-
 to control the browser window size. If the test passes, it means your webpage hides
 the target element when the window is resized to smaller dimensions.
 
-Finally, try updating TestCafe to the latest version to see if the problem persists.
\ No newline at end of file
+Finally, try updating TestCafe to the latest version to see if the problem persists.
+
+### What are the possible reasons for an HTTP request failure? How can I address them?
+
+#### The Web server is not responding
+
+Check if the Web and DNS servers are online and configured to accept requests to this URL.
+
+#### Unstable or improperly configured network connection
+
+* Check the proxy server's, router's and network connection's settings.
+* Connect to a different network.
+* Restart or replace the router.
+* Use VPN.
+* Bypass or change the proxy server.
+
+#### Not enough resources in the container or CI system
+
+* Decrease the [concurrency factor](../documentation/using-testcafe/common-concepts/concurrent-test-execution.md) to `1`.
+* Deploy the application's Web server on a separate machine.
+* Change the container's resource limits.
+* Run tests on a local device outside a container.
+* In a CI service, switch to a pricing plan that offers more resources.
+* Try a different CI service.
```

Comment:
Which problem does it solve?

Reply:
It can solve the problem with hardware configuration/resources. You can try to find a CI provider that offers better hardware, or a provider that is less popular and don't experience performance degradations due to the high amount of users.

Reply:
Sure, I just want to say it's not obvious for customer.